projects/exui-common-lib/src/lib/components/cookie-banner/cookie-banner.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="govuk-cookie-banner " role="region" attr.aria-label="'Cookies on {{appName}}'" *ngIf="isCookieBannerVisible">
+  <div class="govuk-cookie-banner__message govuk-width-container">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-cookie-banner__heading govuk-heading-m">Cookies on {{appName}}</h2>
+
+        <div class="govuk-cookie-banner__content">
+          <p>We use some essential cookies to make this service work.</p>
+          <p>We’d also like to use analytics cookies so we can understand how you use the service and make improvements.</p>
```

Comment:
Did this content come from somewhere - it doesn't read very well.

Reply:
GDS